core/common/src/ContourDisplay.ts

```diff
@@ -0,0 +1,296 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Symbology
+ */
+
+import { compareBooleans, compareNumbers, CompressedId64Set, OrderedId64Iterable } from "@itwin/core-bentley";
+import { ColorDef, ColorDefProps } from "./ColorDef";
+import { LinePixels } from "./LinePixels";
+
+/** JSON representation of the style settings used by either a minor or major contour. */
+export interface ContourStyleProps {
+  color?: ColorDefProps;
+  pixelWidth?: number;
+  pattern?: LinePixels;
+}
+
+/** The style settings used by either a minor or major contour.
+   * @see [[Contour.majorStyle]]
+   * @see [[Contour.minorStyle]]
+   */
+export class ContourStyle {
+  /** Color that a major or minor contour line will use. Defaults to [[ColorDef.black]].*/
+  public readonly color: ColorDef;
```

Comment:
What if anything is the color's transparency used for?

Reply:
@DStradley the transparency is ignored -- right?

Reply:
Did you consider using `RgbColor` instead to remove ambiguity?

Reply:
I'll refer that to @markschlosseratbentley.

Reply:
Yes, I agree. We should make it use `RgbColor` to remove ambiguity.

Reply:
I changed this to use `RgbColor`.